### DISCUSSION TITLE
green squares

### DISCUSSION THREAD

**Discussion by Medeir0s95** [kubilus1/autoortho#230] [General]
I need help. My autortho there is a green squares when i cruse. Some knows can i solve this? 

![image](https://github.com/kubilus1/autoortho/assets/137718884/8f228d8b-2c3e-4530-ac6f-3b6f2ffa19e1)

**Comment by dpdurst** [comment]
Its the nature of getting the tiles and just what happens with stream. I had the same thing today flying through New Mexico and some of the tiles were just a dark green. Ignore it and fly!

**Reply by Medeir0s95** [reply]
Right now my ortho didn't complete download some square in my route. For me, it's a big problem.

**Reply by Medeir0s95** [reply]
![image](https://github.com/kubilus1/autoortho/assets/137718884/9b5e8566-15bd-4e6e-b4ed-f02adc4e8b40)

**Reply by Medeir0s95** [reply]
Could it also be my computer?

i59400F
RTX3060
16Gb RAM 2666
SSD 1tb

**Reply by rayw45** [reply]
I Have just updated my i5 10400F, GTX1080TI 11GB, PC to 32GB RAM form its previous 16GB.  YEs I have found Windows is slicker and I get less of these sometimes annoying problems with X Plane 12. Maybe worth the upgrade as DDR4 memory is not too expensive today. 

I see  you have wisely installed an SSD drive but again and pretty cheap now I have a separate dedicated one for X Plane 12 and a few similar sims which I rarely play these day with XP12 ruling the roost for me.

Also pay careful attention to the order of the lines in the X Plane12/Custom Scenery/scenery_packs.ini file as that order is vitally important as you probably already know but well wroth mentioning it.

**Comment by kubilus1** [comment]
Increase 'max_wait' time in the config.

**Reply by Medeir0s95** [reply]
Where i ajust that??

**Reply by dpdurst** [reply]
In this path -> C:\Users\ (your username)  is a .autoortho file, open it up and find the line called - maxwait= 0.5 and change it to like 2.0 or 3.0 and see if this makes it better

**Reply by Medeir0s95** [reply]
Sure. Thank you.

**Reply by Medeir0s95** [reply]
Could it also be my computer? 

i59400F
RTX3060 
16Gb RAM 2666
SSD 1tb

**Reply by Medeir0s95** [reply]
Not solve. :( I'm sad for that. 
![image](https://github.com/kubilus1/autoortho/assets/137718884/086ba003-1192-443a-ba73-03789a37934a)

**Comment by Medeir0s95** [comment]
Jesus, my scenery not a load. 
![image](https://github.com/kubilus1/autoortho/assets/137718884/40a27d9d-6fed-4df9-841d-c02e9484e28e)

**Comment by betampona** [comment]
Hello, I was one of the first to test AutoOrtho but as I fly exclusively with the B737-800X from zibo, which involves aircraft speeds of 400 to 550 kts, downloading and setting up the scenery was out of date and I found myself with blue tiles or no scenery at all on the first flight.
The speed at which the tiles are downloaded is limited by the tile donor's server.
I would like to know if it is possible to download all the scenery tiles into the cache according to a flight plan, independently of X-plane. When the cache is already filled with scenery tiles, there is no problem with blue tiles or loss of scenery.
Best regards.

**Reply by dpdurst** [reply]
There is no option at this time to download via flight plan. The program only streams while you are flying.

**Comment by betampona** [comment]
Hi, I don't think AutoOrtho is designed for liner theft. You're limited by the download speed of the bing or other servers. To use AutoOrtho with airliners, you need to have the tiles already in the cache, which means you need to download them before the flight. Easy Bing Maps Dowloader delivers free 256x256 .jpeg tiles at ZL13 and up to ZL 20 for a fee.Best regards.envoyé : 1 juillet 2023 à 20:42de : dpdurst ***@***.***>à : kubilus1/autoortho ***@***.***>cc : betampona ***@***.***>, Comment ***@***.***>objet : Re: [kubilus1/autoortho] green squares (Discussion #230) There is no option at this time to download via flight plan. The program only streams while you are flying.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

**Comment by Medeir0s95** [comment]
For update, i change my setup, i put here AMD Ryzen 5950X, 32gb RAM 3600 and my ortho didn't have a bug, sometimes green squares, but for me i imagine that it's a normal. So right now to me, it's so good.

**Comment by Medeir0s95** [comment]
![image](https://github.com/kubilus1/autoortho/assets/137718884/2dc232e0-558d-40bf-9f6a-ddf1eebd7c5f)

**Comment by dpdurst** [comment]
Everyone at some points gets the little green squares, even I do just fly and ignore them or use local Orthophotos and you won't get them.

**Reply by Medeir0s95** [reply]
Yeah. When I do update my PC, the problem it's over, Although continuous green squares in flight. For me isn't a problem.

**Reply by hawajka76** [reply]
What would be the best way currently of combining pre-downloading orthos with Autoortho? Also do you think it would be a lot of work to implement downloading or downloading parts based on flightplan?

**Reply by kubilus1** [reply]
Combining orthos is entirely up to how XPlane handles orthos, so if pre-downloaded stuff is higher in your scenery list, that'll get used.  As for flightplans, that would be a outside the scope of this project.  I think the point of this project is to download based on the flight as you do it.

What you can do though, is generate your own areas via Ortho4XP and just drop that in the z_autoortho/scenery folder.  The doc has some instructions on what config options you want in Ortho4XP.

**Reply by hawajka76** [reply]
"just drop that in the z_autoortho/scenery folder." Like in the attached picture? If there is an alternative to not having to edit the .ini and Custom Scenery folder everytime Ortho4XP is used that would be pretty great. 
![Screenshot from 2023-07-17 10-14-43](https://github.com/kubilus1/autoortho/assets/35002520/ce932102-968f-434b-9a8f-368e0972d737)

**Comment by martinmarino63** [comment]
Hi! Just getting started with ortho4XP and autoortho. May increasing cache size when flying on same areas daily could help reducing green squares?
